src/__internal__/checkable-input/checkable-input.component.js

```diff
@@ -56,6 +56,8 @@ const CheckableInput = ({
     fieldHelp,
     fieldHelpInline,
     helpId,
+    // For radio control, we don't want help icon to be reachable via keyboard navigation
+    helpTabIndex: isRadio ? -1 : undefined,
```

Comment:
Is that conditional necessary here? Correct me if I'm wrong but shouldn't it be just `helpTabIndex: -1` since Checkbox and Switch should behave the same?

Reply:
I have checked the behaviour of `Switch` and `Checkbox` in storybook and with both the help icon can still be reached using the keyboard, so I will make this change 👍